polygon/bor/bor.go

```diff
@@ -1208,8 +1252,18 @@ func (c *Bor) Seal(chain consensus.ChainHeaderReader, blockWithReceipts *types.B
 		}
 	}
 
+	var delay time.Duration
 	// Sweet, the protocol permits us to sign the block, wait for our time
-	delay := time.Until(time.Unix(int64(header.Time), 0))
```

Comment:
[nitpicking] To avoid duplication we could keep this line, but override it if `IsBhilai && successionNumber == 0`